build.cake

```diff
@@ -90,16 +90,24 @@ Task("BuildArtifacts")
                     fileNames.Add(fileName);
                 }
 
-                // DnsClient.dll is needed by Sandcastle
-                if (targetFramework == "net452" && project == "MongoDB.Driver.Core")
+                switch (project)
                 {
-                    fileNames.Add("DnsClient.dll");
+                    case "MongoDB.Driver":
+                         fileNames.Add("mongocrypt.dll");
+                         fileNames.Add("mongocrypt.pdb");
+                         break;
                 }
```

Comment:
One case `switch` looks strange, it could be replaced with `if` safely

Reply:
This section has been removed since we no longer need to generate zip file releases per offline discussion :)